auth_signup_confirmation/data/email.xml

```diff
@@ -0,0 +1,18 @@
+<openerp>
+    <data noupdate="1">
+        <record id="email_registration" model="email.template">
+            <field name="name">email for registation</field>
+            <field name="email_from">${ctx["email_from"]}</field>
+            <field name="subject">Register:</field>
+            <field name="partner_to">${object.partner_id.id}</field>
+            <field name="email_to">${ctx["email_to"]}</field>
+            <field name="model_id" ref="base.model_res_partner"/>
+            <field name="auto_delete" eval="True"/>
+            <field name="body_html"><![CDATA[
+                <p>Registration link:
+                    ${ctx["link"]}
+                </p>
```

Comment:
<p>Follow the link to finish registration:  ${ctx["link"]}
                 </p>